src/wolfgame.py

```diff
@@ -65,6 +65,7 @@
 var.LAST_TIME = None
 var.LAST_START = {}
 var.LAST_WAIT = {}
+var.LAST_GOAT = {}
 
 var.USERS = {}
 
```

Comment:
Also remove every instance that dict is mentioned/used. You missed some spots elsewhere, like in game start.